workers/ohsome_quality_analyst/cli_opts.py

```diff
@@ -5,11 +5,28 @@
 import click
 
 from ohsome_quality_analyst.utils.definitions import (
-    DATASET_NAMES,
+    DATASETS,
     load_layer_definitions,
     load_metadata,
 )
 
+
+class IntOrStrParamType(click.ParamType):
+    """
+    Parse parameter as Interger otherwise parse as String
+
+    https://click.palletsprojects.com/en/8.0.x/parameters/#implementing-custom-types
+    """
+
+    def convert(self, value, param, ctx):
+        try:
+            return int(value)
+        except ValueError:
+            if isinstance(value, str):
+                return value
+        self.fail(f"{0} is not a valid integer or string".format(value), param, ctx)
```

Comment:
Do we need a unittest for this class/line?

Reply:
I added a test case to cover this Class